src/status_im/transport/message/v1/protocol.cljs

```diff
@@ -38,6 +41,26 @@
                                         :topic    topic}
                                        whisper-opts)}]}))
 
+(defn send-direct-message
+  "Sends the payload using to dst"
+  [{:keys [db] :as cofx} dst success-event payload]
+  (let [{:keys [current-public-key web3]} db]
+    {:shh/send-direct-message [{:web3 web3
+                                :success-event success-event
+                                :src     current-public-key
+                                :dst     dst
```

Comment:
why stop use recipients ? what does dst stands for ?

Reply:
destinations, I have aligned it with status-go, but can change, don't mind